jerry-core/parser/js/byte-code.h

```diff
@@ -606,6 +606,8 @@
               VM_OC_LINE) \
   CBC_OPCODE (CBC_EXT_THROW_REFERENCE_ERROR, CBC_NO_FLAG, 1, \
               VM_OC_THROW_REFERENCE_ERROR) \
+  CBC_OPCODE (CBC_EXT_THROW_SYNTAX_ERROR, CBC_HAS_LITERAL_ARG, 1, \
+              VM_OC_THROW_SYNTAX_ERROR | VM_OC_GET_LITERAL) \
```

Comment:
Note: at some point maybe we should merge these throwers. It least on vm level.

Reply:
I can agree with that.